app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -36,6 +36,14 @@ const float kPinchRange = kPinchStart - kPinchThreshold;
 // 0.7 is generally accepted as good for objects facing each other.
 const float kPalmHeadThreshold = 0.7;
 
+// We apply a exponential smoothing filter to the measured distance between index and thumb so we
+// can avoid erroneous click and release events. This value is the smoothing factor of said filter.
+const double mSmoothFactor = 0.5;
```

Comment:
This is the smoothing factor of the filter. Feel free to experiment with different values.

In my tests, a value of 0.5 added a delay of 1–3 frames compared with the previous implementation. This was not noticeable at all.

For comparison, a value of 0.25 added a greater delay (up to 7 frames or so) and was noticeable.

Reply:
We normally use k prefix for constants